Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandMapAreaSelectorView.swift

```diff
@@ -43,10 +43,10 @@ struct OnDemandMapAreaSelectorView: View {
     }
     
     /// The corner radius of the area selector view.
-    private let cordnerRadius: CGFloat = 16
+    private let cornerRadius: CGFloat = 16
```

Comment:
I'd recommend this just being an internal static, that way the handle can access it without having to pass it.